### PR TITLE
BSP-3097, BSP-3101 - Improvements to JSON view response in styleguide

### PR DESCRIPTION
**Pull request by rhino88**

### CODE REVIEW

File: js/example.js [hunk 1]
```diff
@@ -116,7 +116,11 @@ module.exports = function (styleguide, filePath) {
   // Returns all pairs who's keyname doesn't start with an '_'
   function removePrivateKeys (data) {
     return _.omit(data, function (value, key) {
-      return _.startsWith(key, '_')
+      if (Object(value) === value) {
+        removePrivateKeys(value)
+      }
+
+      return _.startsWith(key, '_') && !_.eq(key, '_view')
```
Comment (jpencola) [inline js/example.js]: What do you think about just using `(key !== '_view')` here to keep our usage of lodash to a minimum?

File: js/example.js [hunk 2]
```diff
@@ -116,7 +116,11 @@ module.exports = function (styleguide, filePath) {
   // Returns all pairs who's keyname doesn't start with an '_'
```
Comment (jpencola) [inline js/example.js]: The comment should be re-worded to match the code change

**Review by jpencola — CHANGES_REQUESTED** [review]

File: js/example.js (hunk 1 above)
Comment (rhino88) [reply js/example.js]: Sure no problem, I just tried to use it for consistency

**Comment by rhino88** [comment]
Closing this while I fix a few things

**Pull request closed without merging**